### ISSUE TITLE
Deprecation warning: dbdict.py

### ISSUE THREAD

**Issue by mefuller** [TCKDB/TCKDB#24]
`make test` on clean conda environment install from git repo passes but with following warning:
```
../../../../../miniconda3/envs/tck_env/lib/python3.7/site-packages/requests_cache/backends/storage/dbdict.py:9: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import MutableMapping

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

**Comment by alongd** [comment]
Thanks! An excellent first-issue 👍

**Comment by mefuller** [comment]
Looks like this was fixed [upstream](https://github.com/reclosedev/requests-cache/blob/master/requests_cache/backends/storage/dbdict.py) a while ago, so I guess we just need to bump the version required in the environment?
Or is it automatic that conda knows if the Python version goes to 3.9 that requests-cache needs to get bumped?
How/where do I find the conda package information (requires, provides) and the available versions? I haven't looked too closely, so I wonder why my local version of requests-cache is so out of date.

**Comment by alongd** [comment]
If I understand the question correctly, the env instructions are here:[ environment.yml](https://github.com/TCKDB/TCKDB/blob/master/environment.yml)
We need to see what triggered this warning (`dbdict.py` isn't a TCKDB file)

**Comment by mefuller** [comment]
No, my question was more that given the instructions in environment.yml, how do I see what additional dependencies each package has and how conda solved the environment - I alreadyfound the the file dbdict.py is in requests_cache, but I don't know what requires it and if there are any restrictions on the required version.

Interestingly, it looks like there are conda packages for both requests_cache and requests-cache, which I think are supposed to be the same.

**Comment by alongd** [comment]
I see now. No, I don't know how to find out which dependency calls which. If you happen to find out, I'm happy to learn.
